### ISSUE TITLE
Enhance documentation for supported targets

### ISSUE THREAD

**Issue by DanielMSchmidt** [Galooshi/happo#195]
Hi there,

This seems like an awesome project and I am so excited to try it out, but I only stumbled on its support for React Native only by seeing a related PR. I have the feeling a lot of features are missing in the docs. If this is the case I would be happy to provide a PR.

Regards!

**Comment by trotzig** [comment]
Thanks! I think the reason RN support is not in the README is because @lelandrichardson and others over at Airbnb are still working on making it robust. I don't have full visibility into their plans, perhaps @lencioni can chime in?

**Comment by DanielMSchmidt** [comment]
Oh okay, then maybe adding a beta section to the readme or something similar could be a good idea? Just so that the current status is clearly communicated and I know that I may try it but it also may eventually break.

**Comment by lencioni** [comment]
That's right. @lelandrichardson and @spikebrehm are working on React Native which will be part of the 5.0.0 release.

**Comment by tlvenn** [comment]
Hi, did the RN target make it to the 5.0.0 release ? What is the current status of that target ? Thanks !

**Comment by trotzig** [comment]
Unfortunately, work there was never finished up. I'm mainly maintaining this project on a bug-fix basis. I'm focusing on happo.io, which is a continuation of this project.

**Comment by tlvenn** [comment]
I would be very happy to be a customer of happo.io, does it support RN ?

**Comment by trotzig** [comment]
I'll provide a short answer here and then I'll elaborate in a direct email to you @tlvenn. 

There's no built-in RN renderer, but happo.io has[ an API](https://happo.io/docs/api) that you can use with any platform, as long as you can produce screenshots yourself.

**Comment by tlvenn** [comment]
Thanks @trotzig !
